src/components/Header/Header.module.css

```diff
@@ -35,6 +43,21 @@
     display: none;
 }
 
+.header__search-icon {
+    -webkit-tap-highlight-color: rgba(255, 255, 255, 0) !important;
```

Comment:
!important is also not a good practice

Reply:
Yup, I think I was testing something and forgot to remove them. Done.